Testscripts/Linux/verify_vmbus_interrupts.sh

```diff
@@ -45,12 +48,12 @@ function verify_vmbus_interrupts() {
     #
     # Verify if VMBUS interrupts are processed by all CPUs
     #
-    while read line
+    while read -r line
     do
         if [[ ($line = *hyperv* ) || ( $line = *Hypervisor* ) ]]; then
-            for (( core=0; core<=$cpu_count-1; core++ ))
+            for (( core=0; core<=${cpu_count-1}; core++ ))
```

Comment:
```suggestion
            for (( core=0; core<=$((cpu_count-1)); core++ ))
```

Reply:
Or, alternatively:

```suggestion
            for core in $(seq $((cpu_count - 1)))
```

So that `$((cpu_count - 1))` is evaluated only once.  (Not really necessary, as this is just a shell script and whatnot.)

Reply:
(Otherwise, the loop won't run.)

Reply:
Fixed in 81d60df154909338e515f24ef5140bc7f43e392a. Verified also.